spark-tsfile/src/main/scala/org/apache/iotdb/tsfile/package.scala

```diff
@@ -19,15 +19,23 @@
 
 package org.apache.iotdb
 
-import org.apache.spark.sql.{DataFrame, DataFrameReader, DataFrameWriter}
+import org.apache.spark.sql.{DataFrame, DataFrameReader, DataFrameWriter, SparkSession}
 
 package object tsfile {
-
   /**
     * add a method 'tsfile' to DataFrameReader to read tsfile
     */
   implicit class TsFileDataFrameReader(reader: DataFrameReader) {
-    def tsfile: String => DataFrame = reader.format("org.apache.iotdb.tsfile").load
+    def tsfile(path: String,
+               isNewForm: Boolean = false,
+               spark: SparkSession = null): DataFrame = {
```

Comment:
could the parameter spark be omitted and gotten from inside?

Reply:
spark can't be omitted because if we omit spark, we will get an error when we compile our jar, which is "value spark not found". As for got from inside, I can't get from reader and I'm not sure if I get from context all configuration is same as the client wish